ukol_03/sifra.py

```diff
@@ -0,0 +1,14 @@
+plain_text = input("Please, insert your message to encrypt: ")
+shift = int(input("Please, insert shift: "))
```

Comment:
Tu by sa zišlo nejaké ošetrenie či sa jedná o číslo a nie nejaký nezmysel. Môžeš to overiť napríklad metódou `.isdecimal()` alebo `.isnumeric()`